src/api/app/controllers/webui/project_controller.rb

```diff
@@ -56,24 +56,24 @@ def index
   end
 
   def autocomplete_projects
-    render json: Project.autocomplete(params[:term]).not_maintenance_incident.order(:name).pluck(:name)
+    render json: Project.autocomplete(params[:term]).not_maintenance_incident.limit(50).pluck(:name)
```

Comment:
I think that the limit should also be part of the scope, as it part of the autocomplete feature. Can you move it (everywhere) please please? :pray: